test/uk/gov/hmrc/residencenilratebandcalculator/controllers/NoThresholdIncreaseControllerSpec.scala

```diff
@@ -36,6 +36,10 @@ class NoThresholdIncreaseControllerSpec
 
   val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("", "")
 
+  val userAnswers = mock[UserAnswers]
+
+  val reason = mock[Reason]
```

Comment:
Nit: Since these 2 mocks are only used in a single test, you could create them in the test case itself.